src/main/java/uk/gov/pay/ledger/payout/entity/PayoutEntity.java

```diff
@@ -0,0 +1,134 @@
+package uk.gov.pay.ledger.payout.entity;
+
+import java.time.ZonedDateTime;
+
+public class PayoutEntity {
+
+    private final Long id;
+    private final String gatewayPayoutId;
+    private final Long amount;
+    private final ZonedDateTime createdDate;
+    private final ZonedDateTime paidOutDate;
+    private final String statementDescriptor;
+    private final String status;
+    private final String type;
+    private final Long version;
+
+    public PayoutEntity(Long id, String gatewayPayoutId, Long amount,
```

Comment:
Sorry missed out first time. We could actually pass `PayoutEntityBuilder` so constructor parameters are manageable and is easy to add more values if needed. Usual recommended by lint tools are a max of 7.

Reply:
will do!